python/ray/rllib/rollout.py

```diff
@@ -124,37 +140,46 @@ def rollout(agent, env_name, num_steps, out=None, no_render=True):
     while steps < (num_steps or steps + 1):
         if out is not None:
             rollout = []
-        state = env.reset()
+        obs = env.reset()
+        multi_obs = obs if multiagent else {0: obs}
```

Comment:
could use https://github.com/ray-project/ray/blob/master/python/ray/rllib/env/base_env.py#L196 instead of 0

Reply:
Done.